src/index.html

```diff
@@ -6,13 +6,71 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
```

Comment:
The `preconnect` link to `fonts.gstatic.com` is not necessary if you have already declared `preconnect` to `fonts.googleapis.com`, as it will be covered by the DNS lookup for Google Fonts.